Makefile

```diff
@@ -365,6 +370,18 @@ e2e-test: e2e-bootstrap e2e-helm-deploy # run test for windows
 e2e-teardown: $(HELM)
 	helm delete csi-secrets-store --namespace kube-system
 
+.PHONY: e2e-deploy-manifest
+e2e-deploy-manifest:
+	kubectl apply -f manifest_staging/deploy/csidriver.yaml
+	kubectl apply -f manifest_staging/deploy/rbac-secretproviderclass.yaml
+	kubectl apply -f manifest_staging/deploy/rbac-secretproviderrotation.yaml
+	kubectl apply -f manifest_staging/deploy/rbac-secretprovidersyncing.yaml
+	kubectl apply -f manifest_staging/deploy/secrets-store-csi-driver-windows.yaml
```

Comment:
Same substitution for windows too

Reply:
We are only building linux image for test. Hence skipping for windows